src/Controls/src/Core/Compatibility/Handlers/Shell/Android/ShellToolbarTracker.cs

```diff
@@ -662,6 +661,12 @@ protected virtual void UpdateToolbarItems(AToolbar toolbar, Page page)
 
 				if (SearchHandler.SearchBoxVisibility == SearchBoxVisibility.Collapsible)
 				{
+					var existingItem = menu.FindItem(_placeholderMenuItemId);
+					if (existingItem != null)
```

Comment:
The validation here is fine, but not necessary.
https://learn.microsoft.com/en-us/dotnet/api/android.views.imenu.removeitem?view=net-android-35.0
If there is no item with this identifier, nothing happens.

`menu.RemoveItem(_placeholderMenuItemId);`

Reply:
@jsuarezruiz I have updated the code based on your suggestion. Could you please share your concerns on this.